docs/user_manual/components.md

```diff
@@ -224,6 +224,52 @@ $$
 where $z_{\text{base}} = 1 / y_{\text{base}} = {u_{\text{2, rated}}}^2 / s_{\text{base}}$. 
 Here, $s_{\text{base}}$ is a constant value determined by the solver and $u_{\text{2, rated}}$ is rated voltage at `to_node`.
 
+
+### Generic_Branch  
```

Comment:
our other sections on components consisting of 2 words (e.g. `three_winding_transformer` and `transformer_tap_regulator`) have spaces as a section header
```suggestion
### Generic Branch  
```